pkg/kubelet/kubelet.go

```diff
@@ -1048,7 +1055,16 @@ func (kl *Kubelet) syncPod(pod *api.BoundPod, dockerContainers dockertools.Docke
 			// look for changes in the container.
 			if hash == 0 || hash == expectedHash {
 				// TODO: This should probably be separated out into a separate goroutine.
-				healthy, err := kl.probeLiveness(podFullName, uid, podStatus, container, dockerContainer)
+				ready := probe.Unknown
+				healthy, err := kl.probeContainer(container.LivenessProbe, podFullName, uid, podStatus, container, dockerContainer, probe.Success)
```

Comment:
I would like people to get in the habit of using "live" rather than "healthy", precisely to clarify the distinction between live and ready. Could you please change the name of the variable to "live" here?